Tests/Functional/Twig/PriceExtensionTest.php

```diff
@@ -36,7 +35,7 @@ public function testGetPriceList()
         $extension->setFormatsMap(array_combine($currencies, ['%s EUR', '%s LTL']));
 
         $result = $extension->getPriceList($twig, 1000);
-        $startPrefix = '<span class="currency currency-eur">1.000 EUR</span>';
+        $startPrefix = '<span class="currency currency-eur">1.000,00 EUR</span>';
```

Comment:
That line is a bit odd, because locally tests were failing without it. I'll remove it and see if it work good in travis.